### ISSUE TITLE
kube-green manager arm64 image seems to contain x64-86 binary (0.5.1)

### ISSUE THREAD

**Issue by Ezcyo** [kube-green/kube-green#349]
Hello!
While trying to deploy kube-green manager to our arm64 cluster we noticed a bunch of exec format error despite explicitely pulling the linux/arm64 image (see https://github.com/kube-green/kube-green/pkgs/container/kube-green/98134253?tag=0.5.1).
Having pulled the image locally I see it is appropriately tagged for arm64:
```
        "Architecture": "arm64",
        "Os": "linux",
```
However after exporting loading the image as a tar archive and checking out the appropriate layer, the injected `manager` seems to be an x86_64 binary:
```
manager: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=0uPvMJgps7wzpMaBSE25/UfrOAEYqVOyhfDmveNLE/NsX2bW5JLNQUXKE6weBG/J8FWpz_2QhL_cF5SapBl, with debug_info, not stripped
```
Could this be fixed please?

**Comment by davidebianchi** [comment]
Hi @Ezcyo, thank you for the report. This should be fixed now, can you try the latest version of the image? If it's ok, I will release a new version.

**Comment by Ezcyo** [comment]
Hello! Thanks for your quick response.
Just pulled the latest and it indeed contains the appropriate binary!

**Comment by davidebianchi** [comment]
The fix is released with the version v0.5.2
